karrot/activities/api.py

```diff
@@ -63,10 +64,15 @@ def get_queryset(self):
     def list(self, request, *args, **kwargs):
         queryset = self.filter_queryset(self.get_queryset()) \
             .select_related('about') \
-            .prefetch_related('about__activityparticipant_set', 'about__feedback_given_by')
+            .prefetch_related('about__activityparticipant_set', 'about__feedback_given_by') \
+            .annotate(timezone=F('about__place__group__timezone'))
         feedback = self.paginate_queryset(queryset)
 
-        activities = set(f.about for f in feedback)
+        activities = set()
+        for f in feedback:
+            activity = f.about
```

Comment:
I guess this is needed because `selech_related` doesn't use your custom queryset? And it works for activities that come with conversations because that uses `prefetch_related`?

Reply:
> I guess this is needed because selech_related doesn't use your custom queryset?

Indeed. Which seems a shame, but also how it is.

> And it works for activities that come with conversations because that uses prefetch_related?

Not sure how it relates/impacts conversations, I didn't change anything there. But I guess yeah that prefetch_related bit ensures it has enough information for the serializer?